### DISCUSSION TITLE
Signcleint internal json rpc issue with metamask

### DISCUSSION THREAD

**Discussion by arbaz-meo** [WalletConnect/walletconnect-monorepo#3345] [Developer Support] [answered]
i m using signclient and testing on mumabi test net polygon and facing the issue  in mkaaing the transaction its says internal json rpc issue 

while my code is  

```
 const tx = {
          from:user_address,
          to:global.token_contract,
          data:data,
          gasPrice: "0x029104e28c",
          gasLimit: "0x5208"
        };
       const tx_hash =  await client.request({
          topic:top.topic,
          chainId:"eip155:80001",
          request: {
            method: "eth_sendTransaction",
            params: [tx],
          },
          
        })
```
i m using signclinet 2.9.2 version

**Comment by arbaz-meo** [comment]
@glitch-txs @0xAsimetriq  can you guys help me

**Reply by xzilja** [reply, marked as answer]
rpc errors usually come from miss formatted rpc requests, not related to sign client.

**Reply by arbaz-meo** [reply]
@0xAsimetriq  can you guide me how can i fix this

**Reply by arbaz-meo** [reply]
@0xAsimetriq issue was related to gass estimation so its showing rpc internal error
